src/style.css

```diff
@@ -0,0 +1,104 @@
+@font-face {
+  font-family: Avenir sans-serif;
```

Comment:
```suggestion
  font-family: Avenir, sans-serif;
```
Через кому потрібно записувати